e2e/csi/csi.go

```diff
@@ -34,6 +40,11 @@ func init() {
 	})
 }
 
+const ns = ""
+
+var pluginWait = &e2e.WaitConfig{Interval: 5 * time.Second, Retries: 24} // 2min
```

Comment:
these values aren't that long, but I'm curious if there is a stance on when to mark a test suite as `Slow` I'm not sure we use that flag, and most people testing on a dev cluster will likely only target the suite they want to test specifically, so more of just a general question

Reply:
That's a good question and I don't have a good guideline. All the E2E tests are "slow" relative to a unit test after all.

As far as the dev cluster, we default to `volumes = false` in the Terraform, so this suite won't get run there.